web/edit/db/index.php

```diff
@@ -53,7 +53,7 @@
     
     // Change database user
     if (($v_dbuser != $_POST['v_dbuser']) && (empty($_SESSION['error_msg']))) {
-        $v_dbuser = preg_replace("/^".$user."_/", "", $_POST['v_dbuser']);
+        //$v_dbuser = preg_replace("/^".$user."_/", "", $_POST['v_dbuser']);
```

Comment:
I would suggest to remove the code instead comment it - code history will be provided over github.